### DISCUSSION TITLE
Allow manual trigger flows to be invoked without item selection

### DISCUSSION THREAD

**Discussion by NigmaX** [directus/directus#15794] [Feature Requests]
Currently, a manual flow requires an item to be selected.
It could be nice if we could add an option to the manual trigger to allow a click without any selection.
It will send the collection name and empty array for the keys.

That way we could create util functions based on the collection only

We can add to the trigger options an "item selection required" field which will be "true" by default.

Might also fix things like #15746

**Comment by rijkvanzanten** [comment]
https://github.com/directus/directus/pull/15977 🙂
